index.html

```diff
@@ -12,13 +12,14 @@
     <header>
       <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
+        <img id="menu-icon" class="menu-icon" src="menu.svg" alt="menu"/>
       </button>
-      <nav id="desktop-nav">
-        <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
-          <li>
+      <nav class="main-nav" id="desktop-nav">
+        <img class="exit" src="exit.svg" alt="exit"/>
+        <ul class="ul-menu">
+          <li class="li-menu"><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li class="li-menu"><a class="nav-link" href="#about-me">About</a></li>
+          <li class="li-menu">
```

Comment:
- [ ] Great job for implementing the functionality of the mobile menu 👌💯, but I noticed that the navbar links in the desktop version are disappeared, kindly fix it.

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/175192649-b90a9e1f-ede9-4123-bf02-01a55f751315.PNG" width="500"/> | <img src="https://user-images.githubusercontent.com/45231276/175192690-b0fb791f-ac71-4d49-b62d-14f8706d3df8.PNG" width="500" /> |

- [ ] The design of the mobile menu is not similar to the Figma design, According to the requirements, you should follow the Figma design as here [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471), so please change the `background color` and add a `horizontal line` after each link to look like the Figma design.💯

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/175189538-11308c4d-5f21-4ddf-8b47-ee66162cac86.PNG" width="500" />   | <img src="https://user-images.githubusercontent.com/45231276/175189556-9cf3fa19-147b-446c-8cf3-f26fe5817a5a.PNG" width="500" /> |

- [ ] I noticed also that the color of the menu icon is not similar to Figma. According to the requirements, you should follow the Figma design as here [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471), so please change the color of the icon to look like the Figma design.💯
#### Actually You did a great job✨ but remember that you should follow the requirements
![5](https://user-images.githubusercontent.com/45231276/175194133-08b24b1d-27e0-4b1e-ad81-4f2cb932f28e.PNG)

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/175193930-55ef2810-be07-468d-8457-ad2c2db0214c.PNG" width="500" />   | <img src="https://user-images.githubusercontent.com/45231276/175193978-1ee9e9e0-2fe7-4ec6-8fde-c8ed405e1ea3.PNG" width="500" /> |